portfolio/src/main/webapp/index.html

```diff
@@ -6,13 +6,25 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>Sean Sanchez</h1>
+      <p>Hi, and welcome to my portfolio. I am a first generation Peruvian college student. I grew up on Long Island, NY and am currently attending college at Cornell University in Ithaca, NY.</p>
+      
+      <p>Meet the family!</p>
+      <img src = "/images/family.JPG"/>
```

Comment:
No need to close void elements like IMG per the style guide.

Reply:
Got it